app/meson.build

```diff
@@ -143,6 +143,12 @@ endif
 
 executable('scrcpy', src, dependencies: dependencies, include_directories: src_dir, install: true, c_args: c_args, link_args: link_args)
 
+# Desktop entry file for application launchers
+if host_machine.system() == 'linux'
+    # -> /usr/share/applications/scrcpy.desktop
```

Comment:
(by default `/usr/local/share/applications/scrcpy.desktop`, but you can keep this comment as is, it's explicit enough)